modules/reference/pages/feature/openidConnectServer/examples.adoc

```diff
@@ -51,7 +45,7 @@ Thus, all client data and token status are held in memory, which works for test
 However, if you want to reconfigure the server, the `localStore` might be cleared so it is not suitable to use for production purposes.
 A database must be used instead.
 
-If you use a database, replace the `localStore` section with xref:/docs/ref/feature/#jdbc-4.3.html[JDBC] database configuration.
+If you use a database, replace the `localStore` section with xref:/docs/ref/feature/#jdbc-4.3.adoc[JDBC] database configuration.
```

Comment:
`xref:/docs/ref/feature/#jdbc-4.3.adoc[JDBC]`

should be

`xref:feature/jdbc-4.3.adoc[JDBC]`